modules/PreviewAPI/src/main/java/org/gephi/preview/PreviewModelImpl.java

```diff
@@ -82,8 +80,7 @@ Development and Distribution License("CDDL") (collectively, the
     //Properties
     private PreviewProperties properties;
     //Dimensions
-    private Dimension dimensions;
-    private Point topLeftPosition;
+    private Rectangle _graphicsCanvas;
```

Comment:
Actually it was a mistake to cache the canvas size. It has to be computed each time to get the last info from the graph model.